hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/Compactor.java

```diff
@@ -135,7 +135,7 @@ public CompactionProgress getProgress() {
     /** Min SeqId to keep during a major compaction **/
     public long minSeqIdToKeep = 0;
     /** Total size of the compacted files **/
-    private long totalCompactedFilesSize = 0;
+    public long totalCompactedFilesSize = 0;
```

Comment:
Opening up all this stuff smells.

Reply:
This was the only marked as private here. Unfortunately needed exposing that to `DirectStoreCompactor`.